app/models/comfy/cms/page.rb

```diff
@@ -97,7 +97,7 @@ def published_at
     elsif scheduled?
       scheduled_on
     elsif ever_been_published?
-      revisions.detect { |revision| revision.data[:event].include?('published') }.created_at
+      revisions.detect { |revision| Array(revision.data[:event]).include?('published') }.created_at
```

Comment:
Good catch. Sounds like the `Array()` fix could potentially also change the behaviour depending on what values we can get for `revision.date[:event]`.